pr_testing/run-pr-profiling.sh

```diff
@@ -1,14 +1,23 @@
 #!/bin/bash -ex
 source $(dirname $0)/setup-pr-test-env.sh
 
-PROFILING_WORKFLOWS=$(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||;s|,| |')
+ALLOWED_PROFILING_WORKFLOWS=$(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||;s|,| |')
+
+for PROFILING_WORKFLOW in $PROFILING_WORKFLOWS;do
```

Comment:
please set `PROFILING_WORKFLOWS=${ALLOWED_PROFILING_WORKFLOWS}` if `$PROFILING_WORKFLOWS` is not set